sql/full-text-search/varchar/query-v2/partitioning/many-partition.sql

```diff
@@ -0,0 +1,143 @@
+CREATE TABLE infection_numbers_influenza (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+)
+PARTITION BY LIST (city_code);
+
+CREATE TABLE infection_numbers_influenza_20_01 (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+ALTER TABLE ONLY infection_numbers_influenza ATTACH PARTITION infection_numbers_influenza_20_01 FOR VALUES IN ('20-01');
```

Comment:
Can we use `CREATE TABLE infection_numbers_influenza_20_01 PARTITION OF infection_numbers_influenza FOR VALUES IN ('20-01')`?

Reply:
I've fixed in https://github.com/pgroonga/pgroonga/pull/236/commits/48e6fd67c664f16eb5856c7f01b513278990ffb9.